cypress/e2e/orders/orders-export-to-a-vendor.cy.js

```diff
@@ -36,6 +36,11 @@ describe('orders: export', () => {
   const integartionDescription = 'Test Integation descripton1';
   const vendorEDICodeFor1Integration = getRandomPostfix();
   const libraryEDICodeFor1Integration = getRandomPostfix();
+  let user;
+  let firstPOLNumber;
```

Comment:
'firstPOLNumber' is defined but never used.

Reply:
fixed